src/components/Icon/Icon.tsx

```diff
@@ -5,68 +5,149 @@ import svg4everybody from 'svg4everybody';
 import styles from './Icon.module.css';
 import icons from '../../icons/spritemap/spritemap.svg';
 
-export interface Props {
+interface IconPropsBase {
   /**
    * CSS class names that can be appended to the component.
    */
   className?: string;
   /**
-   * Declares whether the icon is focusable or not for accessibility
+   * Either a fragment of svg elements (g, path, circle, etc.), or a single
+   * svg element. Useful for creating specific icon components.
+   *
+   * @example
+   * function CircleIcon(props: IconProps) {
+   *   return (
+   *     <Icon {...props}>
+   *       <circle cx="50" cy="50" r="50" />
+   *     </Icon>
+   *   )
+   * }
    */
-  focusable?: boolean;
```

Comment:
the only use cases were setting this to `false` but the default was already `false`, so prop didn't appear to be needed 👀  lmk if there are use cases I didn't think of though

Reply:
Maybe ask BM in case there's something we're missing? I'm sure they had a specific use case in mind when they added this prop.

Reply:
per [slack discussion](https://czi-edu.slack.com/archives/C0310HJ9WEN/p1649173028243059), moving forward with removing it 👍